doc/Makefile

```diff
@@ -143,8 +145,12 @@ man: man/isympy.xml
 _build/logo/sympy-notailtext-favicon.ico: logo
 
 logo: src/logo/sympy.svg
-	rm -rf _build/logo
-	mkdir -p _build/logo
-	$(PYTHON) ./generate_logos.py -d
-	@echo
-	@echo "Logo generated."
+	if [ -d "$(LOGOFOLDER)" ]; then \
```

Comment:
This will not rebuild the logo. We should assume that if someone re-executes the build command they are aiming to re-build the docs so everything should be re-built. I am -1 to this change.